subsys/bluetooth/controller/ll_sw/ull.c

```diff
@@ -2072,13 +2072,17 @@ void ull_prepare_dequeue(uint8_t caller_id)
 	void *param_resume_head = NULL;
 	void *param_resume_next = NULL;
 	struct lll_event *next;
+	uint8_t loop = 10U;
```

Comment:
This needs an explanation. Why do we need this loop? Why is the loop value `10`?

Reply:
Added comment in code now.